### ISSUE TITLE
virtualenv ubuntu package needs also installation

### ISSUE THREAD

**Issue by szoupanos** [aiidateam/aiida_core#538]
On a "clean" ubuntu 16 server, I follow the guide as described at
http://aiida-core.readthedocs.io/en/v0.8.1/installation/index.html

Even if the guide proposes to install virtualenv via pip
`$ pip install --user -U setuptools pip wheel virtualenv`
ubuntu complains during its creation
```
aiida@ubuntu-aiida-vm1:~/aiida-code$ virtualenv ~/aiidapy
The program 'virtualenv' is currently not installed. You can install it by typing:
sudo apt install virtualenv
```

The solution seems to be the installation of the ubuntu's virtualenv package
`sudo apt install virtualenv`

And from a first look, after installing the above package, the installation of virtualen via pip is not needed.

**Comment by danielhollas** [comment]
Confirming this for my Linux Mint 18 installation based on Ubuntu 16.

**Comment by sphuber** [comment]
This was fixed in #561
